internal/auth/auth.go

```diff
@@ -19,8 +19,6 @@ import (
 const (
 	audiencePath           = "/api/v2/"
 	waitThresholdInSeconds = 3
-	// SecretsNamespace is the namespace used to set/get values from the keychain.
-	SecretsNamespace = "auth0-cli"
```

Comment:
This single namespace value is no longer sufficient because we will have multiple secret values and need to distinguish between the different types.